node/test/performance-test/src/gen_ref_constants.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2017-2021 Parity Technologies (UK) Ltd.
```

Comment:
```suggestion
// Copyright 2021 Parity Technologies (UK) Ltd.
```

Reply:
I use `file_header.txt`, isn't it correct?